setup.py

```diff
@@ -76,7 +76,7 @@ def read(path):
         doc=['sphinx>=3,<4',
              'crate-docs-theme'],
     ),
-    python_requires='>=3.4',
+    python_requires='>=3.7',
```

Comment:
We decided to still support Python 3.6, when possible. See https://github.com/crate/crash/pull/369.

Reply:
Was seeing that only Python >= 3.7 is mentioned in `classifiers`, but fine to keep it at 3.4.